src/coffee/run.coffee

```diff
@@ -203,6 +203,7 @@ module.exports = class
       .description 'Allows to publish, unpublish or delete (all) products of your SPHERE.IO project.'
       .option '--changeTo <publish,unpublish,delete>', 'publish unpublished products / unpublish published products / delete unpublished products'
       .option '--csv <file>', 'processes products defined in a CSV file by either "sku" or "id". Otherwise all products are processed.'
+      .option '-o, --output <file>', 'Path to the file if the product being process is more than the default which is 100.'
```

Comment:
I think this is still not clear. How about something like

`Optional path to results log output if it is larger than X`